### PR TITLE
Merge master into 5.0.0

### PR DESCRIPTION
**Pull request by cwardgar**
@lesserwhirls I believe I got the merge right, but there were some conflicts that you should be aware of:

CONFLICT (modify/delete): tds/src/main/webapp/godiva2/godiva2.html deleted in HEAD and modified in master.
CONFLICT (modify/delete): docs/website/tds/tutorial/workshop2015.html deleted in HEAD and modified in master.
CONFLICT (modify/delete): docs/website/tds/tutorial/AddingServices.html deleted in HEAD and modified in master.
CONFLICT (modify/delete): docs/website/tds/reference/WMS.html deleted in HEAD and modified in master.
CONFLICT (modify/delete): docs/website/tds/reference/Services.html deleted in HEAD and modified in master.
CONFLICT (modify/delete): docs/website/tds/TDS.html deleted in HEAD and modified in master.

In all cases, doc changes were made to a file on `master` that no longer exists on `5.0.0`. So, I ignored all those changes. You may want to check that corresponding changes have been made to the new set of docs.

### CODE REVIEW

File: CONTRIBUTING.adoc [hunk 1]
```diff
@@ -2,7 +2,7 @@ image::http://www.unidata.ucar.edu/images/logos/thredds_tds-150x150.png[THREDDS]
 
 = Welcome contributors!
 
-First off, thank you for your intrest in contributing to the THREDDS project!
+First off, thank you for your interest in contributing to the THREDDS project!
```
Comment (lesserwhirls) [inline CONTRIBUTING.adoc]: OMG the typos are terrible - my bad!

**Comment by lesserwhirls** [comment]
Once @DennisHeimbigner takes a look and approves, I'll merge.

File: CONTRIBUTING.adoc [hunk 2]
```diff
@@ -93,12 +93,12 @@ If you would like these changes to be stored on your personal remote repository,
...
 `git push me mywork`
 
-Once you are satisified with your work, there is one last step to complete before submitting the pull request - clean up the history.
+Once you are satisfied with your work, there is one last step to complete before submitting the pull request - clean up the history.
 
 [[gh-history-cleanup]]
 === Clean up your git commit history
 
-Commit history can often be full of temporiariy commit messages, of commits with code changes that ultimately didn't make the final cut.
+Commit history can often be full of temporarily commit messages, of commits with code changes that ultimately didn't make the final cut.
```
Comment (DennisHeimbigner) [inline CONTRIBUTING.adoc]: You mean temporary.  Also prefer
    , of commits
=>
  , and of commits

**Comment by cwardgar** [comment]
As usual, this PR will require a merge, not a rebase.

File: CONTRIBUTING.adoc (hunk 2 above)
Comment (lesserwhirls) [reply CONTRIBUTING.adoc]: and/or - best of no worlds. I'll make a quick change.

Comment (lesserwhirls) [reply CONTRIBUTING.adoc]: Sure enough, I could edit and commit directly to your PR branch using the github interface. Slick.

**Comment by lesserwhirls** [comment]
Travis is having some issues. Hopefully this will clear up soon. I'll keep an eye out on things.

**Comment by lesserwhirls** [comment]
w00t - thanks @cwardgar!

**Pull request merged**